source/manual/local-frontend-development.html.md

```diff
@@ -84,10 +62,34 @@ cd /govuk/govuk-docker
 make government-frontend
 
 cd /govuk/government-frontend
-govuk-docker up government-frontend-app-live
+govuk-docker-up app-live
 # You can now view the app on government-frontend.dev.gov.uk
 ```
 
+## Using startup scripts
+
+**NOTE:** this approach only works for frontend development and relies on live APIs. You will need to find and install any dependencies yourself. Consider using the general GOV.UK Docker environment in the first instance.
+
+If you are making changes to certain frontend apps, you can also view these changes by running the application's `./startup.sh` script, if it has one. This example is for [government-frontend], but these instructions apply to other frontend apps.
```

Comment:
Suggest a slight tweak to emphasise that the startup scripts might not be present and that they vary from app to app:

```suggestion
If you are making changes to certain frontend apps you can also view these changes by running the application's `./startup.sh` script - if it has one. This example is for [government-frontend], but these instructions may apply to other frontend apps.
```

Reply:
Sure, I've updated the 3rd commit with your suggestion 👍.